src/objects/zcl_abapgit_object_devc.clas.abap

```diff
@@ -629,8 +629,10 @@ CLASS zcl_abapgit_object_devc IMPLEMENTATION.
     SELECT SINGLE obj_name
            FROM tadir
            INTO lv_object_name
-           WHERE obj_name <> iv_package_name
-           AND   devclass = iv_package_name.
+           WHERE pgmid    =  'R3TR'
+           AND   object   <> 'DEVC'
+           AND   obj_name <> iv_package_name
+           AND   devclass =  iv_package_name.
```

Comment:
If package name = ZNAME and it contains DOMA ZNAME, this will return true, as if the package is empty? Or perhaps I'm just too tired today

Reply:
hm, you are probably right. I really shouldnt't watch soccer and code... I'll fix it sometime later.

Reply:
I dont remember how subpackages are stored, this is another test case